roles/smallfile-bench/templates/smallfilejob.yaml.j2

```diff
@@ -0,0 +1,6 @@
+operation: create
```

Comment:
The biggest problem right now is that we can only pass one operation type to the operator, since a single smallfile run takes only 1 operation type.   Ideally I'd like to be able to define a test matrix by passing lists of parameter values and have the smallfile operator iterate on them.   Do we need to change the operator or smallfile?  In the past Alex C and I have let [Ceph Benchmarking Tool handle this](https://nuget.pkg.github.com/bengland2/cbt/blob/smallfile/example/smallfile-test-set.yaml) - it accepts parameter lists and can invoke smallfile multiple times to implement this.   If it's possible for Ripsaw to do this, then I'd prefer that.

Reply:
Yupp, I've kept it in the phase 2 of the task . I think we can do this, but there will be a limitation. I still need to explore it a bit more.